src/parquet/column/levels.cc

```diff
@@ -36,7 +36,8 @@ void LevelEncoder::Init(Encoding::type encoding, int16_t max_level,
       break;
     }
     case Encoding::BIT_PACKED: {
-      int num_bytes = BitUtil::Ceil(num_buffered_values * bit_width_, 8);
+      int num_bytes =
+          static_cast<int>(BitUtil::Ceil(num_buffered_values * bit_width_, 8));
```

Comment:
it seems not a lot sense here, since later `num_bytes` is passed to `BitWriter` constructor, which accepts it as `int`